lectures/geom_series.md

```diff
@@ -849,12 +849,35 @@ so $\frac{\partial p_0}{\partial r}$ will always be negative.
 Similarly, $\frac{\partial p_0}{\partial g}>0$ as long as $r>g$, $r>0$ and $g>0$ and $x_0$ is positive, so $\frac{\partial p_0}{\partial g}$
 will always be positive.
 
-## Back to the Keynesian multiplier
+## Exercises
 
-We will now go back to the case of the Keynesian multiplier and plot the
-time path of $y_t$, given that consumption is a constant fraction
-of national income, and investment is fixed.
+```{exercise-start}
+:label: geom_ex1
+```
+Consider a dynamic Keynesian multiplier model
+
+$$
+ y_t = c_t + i_t + g_t \ \ \textrm { and } \ \ c_t = b y_{t-1}.
+$$
+
+Assume that $i_t=i_0$ and $g_t=g_0$ for all $t \geq 0$.
+
+Plot the time path of $y_t$ with the following initial values:
 
+```{code-cell} ipython3
+i_0 = 0.3
+g_0 = 0.3
+b = 2/3
+y_init = 0
+T = 100
+```
+
+```{exercise-end}
+```
+
+```{solution-start} geom_ex1
+:class: dropdown
+```
```

Comment:
`caption: "Path of aggregate output tver time"` -> caption: "Path of aggregate output over time" below

Reply:
I don't know why the checker did not show me this, but thanks a lot @shlff.

Reply:
No worries. ^^